week-4-practise/level-2/exercise-1.js

```diff
@@ -9,17 +9,17 @@ We made a start on this for you here:
 */
 
 var bottle = {
-	volume: 0,
-	fill: function(){
-	 	// calling this function should make you bottles volume = 100; 
-	},
-	drink: function(){
-		// calling this function should decrease your bottles volume by 10;
-	},
-	empty: function(){
-		// this function should return true if your bottles volume = 0
-	}
-}
+  volume: 0,
+  fill: function() {
+    this.volume += 100;
+  },
+  drink: function() {
+    this.volume -= 10;
+  },
+  empty: function() {
+    return this.volume === 0;
+  }
+};

```

Comment:
There's a bug here.  What happens when you call `fill` when the bottle isn't empty?  The spec suggests it should make volume = 100 when `fill` is called.

Not bad, though!